.github/workflows/ci.yml

```diff
@@ -30,17 +30,33 @@ jobs:
           ${{ runner.os }}-poetry-
     - name: Install dependencies
       run: |
-        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python
-        $HOME/.local/bin/poetry install
+        pip install poetry
+        poetry install -vv --no-root --no-interaction
+        poetry show -vv
+    - name: Migrate database
```

Comment:
the tests should migrate the DB when they start, could you explain why we need to migrate with `TESTING: false` here?

Reply:
I was running into a bunch of issues where the ci pipeline would fail during the tests b/c of the newest migration that didn't happen (the alias table didn't exist). I'm pushing a commit to remove this and see if it still happens, since I may have changed some other things that affected it to begin with.

Reply:
okay it looks like it passed, I think I added all this when there was some other underlying problem that I ended up fixing later.

Reply:
oh wait, I see you merged this to master yesterday, I was confused. lol, okay let me answer your original question. When TESTING was true it did this thing where it prepended `test_` onto the database name which I didn't want

Reply:
I think we don't need to run this "Migrate database" step here. The migrations should run fine during the conftest app setup. While testing this I think I found a bug?

I tested your branch both locally and in github and the aliases unit tests are randomly failing or succeeding for me. I suspect the problem comes from `load_modules` [here](https://github.com/uc-cdis/metadata-service/blob/955a293/src/mds/main.py#L94); I haven't had time to try to reproduce it consistently but it looks like if the "aliases" module is loaded before the "maintain" module then the API calls go to `POST /metadata/<path>` instead of `POST /metadata/<path>/aliases`

Reply:
locally, forcing the "aliases" module to load first seems to fix the issue, but in github the module isn't loaded at all: 🤯 
```
Loaded module: %s agg_mds
Loaded module: %s index
Loaded module: %s maintain
Loaded module: %s objects
Loaded module: %s query
```
i've been testing in [this branch](https://github.com/uc-cdis/metadata-service/compare/feat/alias...pauline-migrate-debug)

Reply:
:eyes:

Reply:
Okay I fixed it by making the order of the modules deterministic and guaranteed to not be duplicated (using a set then sorting it). Which I'm pretty sure solves it for this problem, but if we ever need a module to load after a different one, I think we'll need to define a new "plugin" and load from that after this list. Or we just get rid of the plugin module loading all together, I mean I think the idea was to eventually allow configurable install of only certain portions but I don't see us ever really utilizing that.